docs/human-interface-guidelines.md

```diff
@@ -965,20 +965,26 @@ Used for labels and descriptive text.
 
 ### Title Case {#title-case}
 
-Title case means you capitalize like a book or article title.
+Title case means you capitalize like a book or article title. In elementary OS, we follow the [AP Style](https://www.bkacontent.com/how-to-correctly-use-apa-style-title-case/).
+
+- Capitalize the first and last words.
+- Capitalize all nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions (as, because, although).
+- Capitalize all words that are more than three letters long.
+- Don’t capitalize articles, prepositions or conjunctions that are three or fewer letters long.
 
-Capitalize the first and last words.
-All nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions (as, because, although) are capitalized.
 Used for titles, buttons, menus, and most other widgets.
 
 * ex. "Open File" title in a dialog.
 * ex. "Delete 13 Songs" on a button.
-* ex. "Search Contacts..." in a search bar.
+* ex. "Search Contacts" in a search bar.
+* ex. "No Videos Open" on a welcome screen.
```

Comment:
I thought about that but was having a hard time thinking of a frequent case like that.